website/content/docs/ecs/requirements.mdx

```diff
@@ -1,11 +1,11 @@
----
+"---
 layout: docs
-page_title: Requirements - AWS ECS
+page_title: Requirements - Consul on AWS Elastic Container Service (ECS)
 description: >-
-  Requirements for Consul Service Mesh on AWS ECS (Elastic Container Service).
----
+  Consul has requirements to install and run on Amazon Web Services ECS. Learn about Consul's requirements for Fargate and EC2, including network mode and subnet information, as well as server, routing, and ACL controller considerations to keep in mind.
+---"
```

Comment:
intended quote?

Reply:
Probably not. I think Jeff copied and pasted from an external draft that mistakenly included quotation marks.